app/components/ToDos.js

```diff
@@ -12,53 +12,38 @@ import { CheckBox, Icon } from "react-native-elements";
 import { connect } from "react-redux";
 
 import TodosQuery from "../redux/queries/todos"
-import { addTodo, completeTodo } from "../redux/actions/todos";
+import { addTodo, completeTodo, deleteTodo } from "../redux/actions/todos";
 
 //type Props = {};
 class ToDos extends Component<Props> {
 
   componentDidMount(){
-    this.props.addTodo({
+    const { addTodo } = this.props; 
+    addTodo({
       id: 1,
       toDo: "Build a Todo App",
       completed: true
     });
-    this.props.addTodo({
+    addTodo({
       id: 2,
       toDo: "Do presentation",
       completed: false
     });
-    this.props.addTodo({
+    addTodo({
       id: 3,
       toDo: "Learn react native",
       completed: false,
     })
   }
 
-  handleChecked = id => {
-    const markToDo = this.props.todos.map(item => {
-      if (item.id == id) {
-        item.completed = !item.completed;
-        return item;
-      } else {
-        return item;
-      }
-    });
-    this.setState({ toDos: markToDo });
-  };
-
-  handleDelete = id => {
-    const filterById = this.props.todos.filter(item => item.id !== id);
-    this.setState({ toDos: filterById });
-  };
-
   render() {
+    const { todos, deleteTodo, completeTodo } = this.props
     return (
       <View style={styles.container}>
         <Text style={styles.welcome}>ToDo</Text>
         <View>
           <FlatList
-            data={this.props.todos}
+            data={todos}
             renderItem={({ item }) => {
               const todo = item
```

Comment:
You could do some handy destructuring here, which would allow you to get rid of `todo`

```javascript
renderItem={({ item: { completed, id, toDo } }) => {
```

This would also mean you have to get rid of all the references to `toDo` in your `renderItem` function.